textproc/lowdown/Portfile

```diff
@@ -1,10 +1,8 @@
 PortSystem		1.0
-PortGroup		snowleopard_fixes 1.0
 
 name			lowdown
 categories		textproc
-version			0.3.1
-revision		1
+version			0.3.2
 license			BSD
 
```

Comment:
Indentation on "version" and "license" is wrong, and should be fixed. Are there tabs in use here? If so, see: https://guide.macports.org/#development.practices.portstyle — tabs violate the style guide.